ddp-workspace/projects/ddp-dsm-ui/src/app/FON/components/input-field/input-field.component.ts

```diff
@@ -103,14 +102,14 @@ export class InputFieldComponent implements OnInit, ControlValueAccessor {
   }
 
   public setValue(inputElement: HTMLInputElement): void {
-    this.value = inputElement.value;
+    this.value = inputElement.value.trim();
     this.onChange(this.value);
     this.onTouched();
   }
 
   public get getErrorMessage(): string {
     if (this.formControl.hasError('required')) {return 'You must enter a value';}
-    return this.formControl.hasError('email') ? 'Not a valid email' : '';
+    return this.formControl.hasError('pattern') ? 'Not a valid email' : '';
```

Comment:
may be `pattern` -> `emailPattern` ?

Reply:
That depends on this property:
![image](https://user-images.githubusercontent.com/77500504/176395554-3b25214d-cb23-4094-8f68-0ff4d1c5ad05.png)
Therefore, the property name derives from the validator function.
I will improve that later, by creating a custom validator.

Reply:
Done